### PR TITLE
Feature/subcontent  color

### PR DESCRIPTION
**Pull request by Snugug**
Also fixed some indentation in the Animation guideline

### CODE REVIEW

**Comment by Snugug** [comment]
Anyone want to :+1: this? Or should I just merge it in?

File: library/templates/subcontent/_color.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        {% endfor %}
+      </ul>
+      {% endif %}
+    </div>
+  </div>
+{% endmacro %}
+
+{% macro swatch(color) %}
+  <div library-class="color-swatch" style="background: {{ color.palette|ibmHex(color.tone) }}; color: {{color.tone|ibmTextColor }};">
+    <p library-class="color-swatch--name">{{ color.palette}} {{ color.tone }}</p>
```
Comment (ryanbrownhill) [inline library/templates/subcontent/_color.html]: should there be a space after`color.palette` in the brackets like --> `{{ color.palette }}`?

Comment (Snugug) [inline library/templates/subcontent/_color.html]: Yup, probably. If you want to comment on a specific line, you can hover over the by the line number and click the `+` to add a line note

**Comment by ryanbrownhill** [comment]
:+1: space fixed, looks good to me

**Comment by britanyponvelle** [comment]
Spaces woo! :+1:

**Pull request merged**